src/lib/features/feature-toggle/feature-toggle-strategies-store.ts

```diff
@@ -922,7 +926,8 @@ class FeatureStrategiesStore implements IFeatureStrategiesStore {
             const featureRow = acc[row.feature_name];
             if (
                 featureRow.lastSeenAt === undefined ||
-                new Date(row.env_last_seen_at) > new Date(featureRow.lastSeenAt)
+                new Date(row.env_last_seen_at) >
+                    new Date(featureRow.last_seen_at)
```

Comment:
We didn't have this property on the row

Reply:
Damn, that was my typo. Good catch.

Reply:
Yeah, small typo. But the row itself also didn't have last_seen_at. I had to add it to the select columns also for this to work.